elliott/elliottlib/errata.py

```diff
@@ -591,32 +591,6 @@ def get_advisory_builds(advisory, session=None):
     return advisory_builds
 
 
-def get_advisory_nvrs(advisory):
-    """
-    :return: dict, with keys as package names and values as strs in the form: '{version}-{release}'
```

Comment:
This assumes that one package name can have only build, which is not true when we could have rhel8 and rhel9 builds at the same time. Remove this